language-support/ts/codegen/tests/ts/build-and-lint/src/__tests__/test.ts

```diff
@@ -6,11 +6,11 @@ import waitOn from 'wait-on';
 import { encode } from 'jwt-simple';
 import Ledger, { CreateEvent, ArchiveEvent, Event, Stream } from  '@daml/ledger';
 import pEvent from 'p-event';
-import * as Main from '../../../daml2ts/build-and-lint-pkg/lib/Main';
-import * as LibMod from '../../../daml2ts/build-and-lint-pkg/lib/Lib/Mod';
 
-const LEDGER_ID = 'daml2ts-tests';
-const APPLICATION_ID = 'daml2ts-tests';
+import * as buildAndLint from '@daml.js/build-and-lint-1.0.0'
```

Comment:
How far are we away from importing it as
```suggestion
import buildAndLint from '@daml.js/build-and-lint-1.0.0'
```

Reply:
i don't know. experimentation required.

Reply:
oh wow. i'm so sorry,
> Ideally, we'd have only one import `import foo from "@daml2ts/foo"`

i don't know how i managed to look past that in the spec 😞 

the good news is i appear to have a solution that provides the ideal syntax:
```typescript
import myProj from '@daml.js/myproj-0.0.1'

test('Use my-project', () => {
  const gift: myProj.Main.Give = {newOwner: 'grover'}
  console.log('Gift : ' + gift);
  console.log('Packge id : ' + myProj.packageId);
  const foo: myProj.A.B.Get_foo = {foo: "bar"}
  console.log ('Foo : ' + foo)
});
```

Reply:
https://github.com/digital-asset/daml/pull/4946